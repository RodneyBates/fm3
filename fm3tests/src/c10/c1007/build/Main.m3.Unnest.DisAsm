 ItkEOF()
 ItkRightEnd()
 ItkBlockBEGINPatch(C281,L1(16_1),[26,2])
 ItkBlockRt(L1(16_1),[13,2])
 ItkStmtListLt(L0(16_0),[27,2])
 ItkStmtListRt(L0(16_0),[27,2])
 ItkBlockDeclListLtPatch(C9,L6(16_6),[13,2])
 ItkBlockDeclListRt(L6(16_6),[13,2])
 ItkBlockDeclListSepPatch(C133,L3(16_3),[19,0])
 ItkBlockDeclListSepPatch(C228,L2(16_2),[23,2])
 ItkTypeDeclLtPatch(C228,[24,4])
 ItkTypeDeclEqualsPatch(C234,[24,7])
 ItkTypeDeclRt([24,4])
 ItkReservedId(Rid"BOOLEAN",[24,9])
 ItkDeclId(L8(16_8),Id7"U1",[24,4])
 ItkSkipLtPatch(C209,L4(16_4))
 ItkSkipRt(L4(16_4))
 ItkReservedId(Rid"CHAR",[23,9])
 ItkDuplDeclId(Id5"T1",[23,4])
 ItkBlockDeclListSepPatch(C174,L1(16_1),[21,2])
 ItkTypeDeclLtPatch(C174,[22,4])
 ItkTypeDeclEqualsPatch(C180,[22,7])
 ItkTypeDeclRt([22,4])
 ItkReservedId(Rid"REAL",[22,9])
 ItkDeclId(L8(16_8),Id6"V1",[22,4])
 ItkSkipLtPatch(C161,L3(16_3))
 ItkSkipRt(L3(16_3))
 ItkIdRefAtom(Id2"T",[21,14])
 ItkTypeDeclLtPatch(C133,[20,4])
 ItkTypeDeclEqualsPatch(C139,[20,7])
 ItkTypeDeclRt([20,4])
 ItkReservedId(Rid"INTEGER",[20,9])
 ItkDeclId(L8(16_8),Id5"T1",[20,4])
 ItkBlockDeclListSepPatch(C98,L2(16_2),[16,0])
 ItkTypeDeclLtPatch(C98,[17,7])
 ItkTypeDeclEqualsPatch(C104,[17,9])
 ItkTypeDeclRt([17,7])
 ItkReservedId(Rid"BOOLEAN",[17,11])
 ItkDeclId(L8(16_8),Id4"U",[17,7])
 ItkSkipLtPatch(C79,L2(16_2))
 ItkSkipRt(L2(16_2))
 ItkReservedId(Rid"CHAR",[16,11])
 ItkDuplDeclId(Id2"T",[16,7])
 ItkBlockDeclListSepPatch(C47,L1(16_1),[14,0])
 ItkTypeDeclLtPatch(C47,[15,7])
 ItkTypeDeclEqualsPatch(C53,[15,9])
 ItkTypeDeclRt([15,7])
 ItkReservedId(Rid"REAL",[15,11])
 ItkDeclId(L8(16_8),Id3"V",[15,7])
 ItkSkipLtPatch(C35,L1(16_1))
 ItkSkipRt(L1(16_1))
 ItkIdRefAtom(Id2"T",[14,17])
 ItkTypeDeclLtPatch(C9,[13,7])
 ItkTypeDeclEqualsPatch(C15,[13,9])
 ItkTypeDeclRt([13,7])
 ItkReservedId(Rid"INTEGER",[13,11])
 ItkDeclId(L8(16_8),Id2"T",[13,7])
 ItkBlockLt(L1(16_1),[13,2])
 ItkLeftEnd()
 ItkBOF()
     0
