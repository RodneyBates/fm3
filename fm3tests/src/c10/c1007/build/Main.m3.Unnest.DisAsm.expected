   323 16_00000000000000a2                   162 ItkEOF()
   321 16_00000000000000a4                   164 ItkRightEnd()
   319 16_00000000000000f0                   240 ItkBlockBEGINPatch(C297,L1(16_1),[24,2])
   312 16_00000000000000ed                   237 ItkBlockRt(L1(16_1),[11,2])
   307 16_00000000000001cc                   460 ItkStmtListLt(L0(16_0),[25,2])
   302 16_00000000000001cf                   463 ItkStmtListRt(L0(16_0),[25,2])
   297 16_00000000000001c6                   454 ItkBlockDeclListLtPatch(C24,L6(16_6),[11,2])
   291 16_00000000000001c7                   455 ItkBlockDeclListRt(L6(16_6),[11,2])
   286 16_00000000000001ca                   458 ItkBlockDeclListSepPatch(C149,L3(16_3),[17,2])
   279 16_00000000000001ca                   458 ItkBlockDeclListSepPatch(C244,L2(16_2),[22,4])
   272 16_00000000000000f5                   245 ItkTypeDeclLtPatch(C244,[22,4])
   266 16_00000000000000f9                   249 ItkTypeDeclEqualsPatch(C250,[22,7])
   260 16_00000000000000f6                   246 ItkTypeDeclRt([22,4])
   256 16_00000000000000b8                   184 ItkReservedId(Rid"BOOLEAN",[22,9])
   250 16_00000000000000ba                   186 ItkDeclId(L8(16_8),Id7"U1",[22,4])
   244 16_00000000000000a9                   169 ItkSkipLtPatch(C225,L4(16_4))
   239 16_00000000000000aa                   170 ItkSkipRt(L4(16_4))
   236 16_00000000000000b8                   184 ItkReservedId(Rid"CHAR",[21,9])
   230 16_00000000000000b9                   185 ItkDuplDeclId(Id5"T1",[21,4])
   225 16_00000000000001ca                   458 ItkBlockDeclListSepPatch(C190,L1(16_1),[20,4])
   218 16_00000000000000f5                   245 ItkTypeDeclLtPatch(C190,[20,4])
   212 16_00000000000000f9                   249 ItkTypeDeclEqualsPatch(C196,[20,7])
   206 16_00000000000000f6                   246 ItkTypeDeclRt([20,4])
   202 16_00000000000000b8                   184 ItkReservedId(Rid"REAL",[20,9])
   196 16_00000000000000ba                   186 ItkDeclId(L8(16_8),Id6"V1",[20,4])
   190 16_00000000000000a9                   169 ItkSkipLtPatch(C177,L3(16_3))
   185 16_00000000000000aa                   170 ItkSkipRt(L3(16_3))
   182 16_00000000000000bc                   188 ItkIdRefAtom(Id2"T",[19,14])
   177 16_00000000000000f5                   245 ItkTypeDeclLtPatch(C149,[18,4])
   171 16_00000000000000f9                   249 ItkTypeDeclEqualsPatch(C155,[18,7])
   165 16_00000000000000f6                   246 ItkTypeDeclRt([18,4])
   161 16_00000000000000b8                   184 ItkReservedId(Rid"INTEGER",[18,9])
   155 16_00000000000000ba                   186 ItkDeclId(L8(16_8),Id5"T1",[18,4])
   149 16_00000000000001ca                   458 ItkBlockDeclListSepPatch(C114,L2(16_2),[15,2])
   142 16_00000000000000f5                   245 ItkTypeDeclLtPatch(C114,[15,7])
   136 16_00000000000000f9                   249 ItkTypeDeclEqualsPatch(C120,[15,9])
   130 16_00000000000000f6                   246 ItkTypeDeclRt([15,7])
   126 16_00000000000000b8                   184 ItkReservedId(Rid"BOOLEAN",[15,11])
   120 16_00000000000000ba                   186 ItkDeclId(L8(16_8),Id4"U",[15,7])
   114 16_00000000000000a9                   169 ItkSkipLtPatch(C95,L2(16_2))
   109 16_00000000000000aa                   170 ItkSkipRt(L2(16_2))
   106 16_00000000000000b8                   184 ItkReservedId(Rid"CHAR",[14,11])
   100 16_00000000000000b9                   185 ItkDuplDeclId(Id2"T",[14,7])
    95 16_00000000000001ca                   458 ItkBlockDeclListSepPatch(C62,L1(16_1),[13,2])
    89 16_00000000000000f5                   245 ItkTypeDeclLtPatch(C62,[13,7])
    84 16_00000000000000f9                   249 ItkTypeDeclEqualsPatch(C68,[13,9])
    78 16_00000000000000f6                   246 ItkTypeDeclRt([13,7])
    74 16_00000000000000b8                   184 ItkReservedId(Rid"REAL",[13,11])
    68 16_00000000000000ba                   186 ItkDeclId(L8(16_8),Id3"V",[13,7])
    62 16_00000000000000a9                   169 ItkSkipLtPatch(C50,L1(16_1))
    58 16_00000000000000aa                   170 ItkSkipRt(L1(16_1))
    55 16_00000000000000bc                   188 ItkIdRefAtom(Id2"T",[12,17])
    50 16_00000000000000f5                   245 ItkTypeDeclLtPatch(C24,[11,7])
    45 16_00000000000000f9                   249 ItkTypeDeclEqualsPatch(C30,[11,9])
    40 16_00000000000000f6                   246 ItkTypeDeclRt([11,7])
    36 16_00000000000000b8                   184 ItkReservedId(Rid"INTEGER",[11,11])
    30 16_00000000000000ba                   186 ItkDeclId(L8(16_8),Id2"T",[11,7])
    24 16_00000000000000ea                   234 ItkBlockLt(L1(16_1),[11,2])
    19 16_000000000000016f                   367 ItkExportIdListRt(L1(16_1),[9,7])
    14 16_0000000000000173                   371 ItkExportIdListElem(Id1"Main",[9,7])
     9 16_000000000000016c                   364 ItkExportIdListLt(L1(16_1),[9,7])
     4 16_00000000000000a3                   163 ItkLeftEnd()
     2 16_00000000000000a1                   161 ItkBOF()
     0
